runtime/commonMain/src/kotlinx/io/Streams.kt

```diff
@@ -16,11 +16,15 @@
 
 package kotlinx.io
 
+import kotlinx.serialization.*
+
+@Deprecated(message = message, level = DeprecationLevel.ERROR)
 expect open class IOException: Exception {
     constructor()
     constructor(message: String)
 }
 
+@InternalSerializationApi
```

Comment:
It should also be deprecated then, isn't it? We plan to remove all this stuff in favor of real `kotlinx-io`

Reply:
I will do it in a follow-up PR. It requires a lot of rewriting in CBOR and ProtoBuf

Reply:
We can still use deprecated classes, so no need to rewriting

Reply:
That's why it's marked as `@InternalSerializationApi` in the first place.